pkg/cmd/codespace/list.go

```diff
@@ -59,6 +67,11 @@ func (a *App) List(ctx context.Context, limit int, exporter cmdutil.Exporter) er
 		tp.AddField("BRANCH", nil, nil)
 		tp.AddField("STATE", nil, nil)
 		tp.AddField("CREATED AT", nil, nil)
+
+		if hasNonProdVSCSTarget {
+			tp.AddField("VSCS TARGET", nil, nil)
```

Comment:
I like that we are filtering this column for users with non-prod codespaces. But note that we probably need to do some filtering here: https://github.com/cli/cli/blob/cd8d6532850aa099da04c0974d4a959a38a85738/internal/codespaces/api/api.go#L223

The list command can be used to return codespaces as json for scripting purposes and by adding the new field to the model it will show up on there unless we also filter it in the `ExportData` method.

Reply:
Thanks for calling that out - I wasn't aware of that behavior. I think that in `ExportData` we won't be able to do this `hasNonProdVSCSTarget` check since we only have a single codespace at a time - instead we can just not include the value if it's production

Reply:
![image](https://user-images.githubusercontent.com/5447118/157915015-a3d3cdc4-df26-42f1-8472-05743669cb26.png)